src/containerapp/azext_containerapp/_params.py

```diff
@@ -119,6 +119,7 @@ def load_arguments(self, _):
     with self.argument_context('containerapp create') as c:
         c.argument('traffic_weights', nargs='*', options_list=['--traffic-weight'], help="A list of revision weight(s) for the container app. Space-separated values in 'revision_name=weight' format. For latest revision, use 'latest=weight'")
         c.argument('workload_profile_name', options_list=['--workload-profile-name', '-w'], help="Name of the workload profile to run the app on.", is_preview=True)
+        c.argument('secret_volume_mount', options_list=['--secret-volume-mount'], help="Path to mount all secrets e.g. mnt/secrets", is_preview=True)
```

Comment:
```suggestion
        c.argument('secret_volume_mount', help="Path to mount all secrets e.g. mnt/secrets", is_preview=True)
```

Reply:
Please do not explicitly define the option name repeatedly

Reply:
thank you, took out in latest commit